docs/synthetics/alerts.md

```diff
@@ -0,0 +1,27 @@
+title: Creating Alerts 
+description: Step by step guide for creating Synthetics Alerts
+
+Synthetics App support 2 types of alerts:
+
+* Synthetics Alerts - These alerts are raised when a run fails based on the configured conditions and rule is auto-created while creating the monitor.
+* Metric Alerts - These are threshold and anomaly alerts that can be created on Synthetics metrics.
+
+## Synthetics Alerts
+
+Synthetics alerts are the alerts to notify the failure of a monitor. They are auto-created while creating the monitor. As of the monitor creation, you could specify the list of conditions that have to be met for the monitor to pass. If any of these conditions fail, then the run will be declared failed and a Synthetics alert will be raised. The conditions are different for [HTTP](/synthetics/http-monitor/#conditions) and [Browser](synthetics/browser-monitor/#conditions) monitors.
+
+You can customize the notification settings & schedule while creating the monitor. You can disable notifications temporarily by turning off `Enable Notifications`. When you turn off the notifications, the monitor will continue to run, but no notifications will be sent on failure. 
+
+You could control when the alert should be triggered based on the consecutive run failures. By default, the alert will be triggered when the monitor fails from any of the locations. You could change the below parameter to control this behavior. For example, by setting this to 2 for a monitor that runs every 10 mins from Frankfurt and Mumbai, the alert will be trigger only when the monitor fails twice consecutively from Frankfurt or Mumbai.
+
+![Run count based alerting](../images/synthetics/create-monitor-run-count-alert.png)
+
+## Metric Alerts
+
+Apart from the auto-created Synthetics alerts, you could create a custom alert on any of the Synthetics metrics and get notified when they are triggered. This could be used to set performance budgets for any of the metrics like page size, resource count, etc. You could create a threshold or anomaly alert. Metric alerts can be created directly from Synthetics charts in the monitor overview page or from custom charts for Synthetics metrics in [dashboard](/dashboards). You can select the metric to create the alert rule by clicking the bell icon when you hover over the chart.
```

Comment:
you could => you can (in 2 places)
Synthetics metrics -- do we have a page that lists all Synthetics metrics?  Should we?  Aha, I see you added that in this PR. :). I think we can link "Synthetics metrics" here to that section.

You can select the metric to create the alert rule by clicking the bell icon when you hover over the chart. -- this might change soon...

Reply:
> this might change soon
I assume you are talking about recent change to add bell icon at report level where user can select the metric from alert create popup. If yes, this feature is not planned for Synthetics screens